src/main/kotlin/controllers/MainController.kt

```diff
@@ -114,6 +127,8 @@ class MainController constructor(
                     addActionListener(::aboutClicked)
                 }.let(::add)
             }.let(::add)
+
+            Box.createHorizontalGlue().let(::add)
```

Comment:
Might want to put this in the check-for-updates function? Right now it looks quite out of place